### ISSUE TITLE
No logging in indicator 

### ISSUE THREAD

**Issue by karek314** [LiskHQ/lisk-ui#12]
Sometimes when click Sign In at https://login.lisk.io/ Logging indicator does not appears in case logging in takes long time it's confusing.

**Comment by fix** [comment]
what do you mean, should we make some animated gif and disabling the logging in button, or were you thinking about something else?

**Comment by karek314** [comment]
Yes, exactly. Possibly animation with disappearing login field and appearing wallet interface.
